go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/kr/text v0.2.0 // indirect
 	github.com/stretchr/testify v1.7.0
 	golang.org/x/crypto v0.0.0-20210921155107-089bfa567519
-	golang.org/x/sys v0.0.0-20211007075335-d3039528d8ac // indirect
+	golang.org/x/sys v0.0.0-20211019181941-9d821ace8654 // indirect
```

Comment:
why is this update needed ? If we update, it should be at the latest version (and we could update other dependencies as well).

Reply:
TODO: revert changes and do in different PR.

Reply:
Done